projects/picker/package.json

```diff
@@ -24,9 +24,9 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@angular/common": "^13.0.3 || ^14.0.0 || ^15.0.0 || ^16.0.0",
-    "@angular/core": "^13.0.3 || ^14.0.0 || ^15.0.0 || ^16.0.0",
-    "@angular/cdk": "^13.0.3 || ^14.0.0 || ^15.0.0 || ^16.0.0"
+    "@angular/common": "^17.0.0",
```

Comment:
Do we want to advertise as `>=17.0.0`? Libraries are generally forward-compatible, and changing this to `>=` would make things easier for consumers of the library going forward.

Reply:
So, I meant literally `>=17.0.0` instead of `^17.0.0`

This would allow the library to be used with Angular 18 when it's released, with no changes to the library.